Source/ExtensionMethods/RectangleFExtensions.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Drawing.Drawing2D;
+using System.Text;
+
+namespace Svg.ExtensionMethods
+{
+    internal static class RectangleFExtensions
+    {
+        internal static PointF[] GetPoints(this RectangleF rectangle)
+        {
+            return new[]
+            {
+                rectangle.Location,
+                rectangle.Location + rectangle.Size
```

Comment:
here should consider five points like this, If there are only two points, there will have issue with the bounds when the transform contains rotation
``` c#
 return new[]
            {
                rectangle.Location,
                rectangle.Location + rectangle.Size,
                new PointF(rectangle.X,rectangle.Y+rectangle.Height),
                new PointF(rectangle.X+rectangle.Width,rectangle.Y)
            };
```

Reply:
you can verify this by this transform : "matrix(0.51 0.51 -0.51 0.51 776.90 96.03)"